cmd/kubelet/app/server_linux.go

```diff
@@ -37,7 +37,7 @@ func watchForLockfileContention(path string, done chan struct{}) error {
 	go func() {
 		select {
 		case ev := <-watcher.Event:
-			klog.InfoS("inotify event", "event", ev)
+			klog.InfoS("Inotify event", "event", ev)
 		case err = <-watcher.Error:
 			klog.ErrorS(err, "inotify watcher error")
```

Comment:
should be capitalized for this line?

Reply:
According to https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
Capitalize the first letter of the sentence